roles/oneagent/tests/requirements.txt

```diff
@@ -1,4 +1,3 @@
-ansible >= 2.15
```

Comment:
You do not have to remove this dependency from `requirements.txt`

with such command 
` pip3 install -r requirements.txt "ansible==9.13.0"` it installs ansible with requested version (as long as this version satisfy >=2.15 requirement)